src/libraries/System.IO.FileSystem/tests/VirtualDriveSymbolicLinks.Windows.cs

```diff
@@ -0,0 +1,292 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+
+namespace System.IO.Tests
+{
+    // Need to reuse the same virtual drive for all the test methods.
+    // Creating and disposing one virtual drive per class achieves this.
```

Comment:
If instead of doing this, I splitted the virtual drive tests into multiple different test classes, then each test class would create its own virtual drive. We have so many symlink tests now, that we would end up depleting the available drive letters.